docs/ko/translation-guide.md

```diff
@@ -302,9 +302,9 @@ A value of 0.01 was used for the value to ramp down to in the last function rath
 
 | 용어 | 번역 | 참고 링크 |
 | --- | --- | --- |
-| Compatibility notes | 호환성 참고 사항 | [링크](https://developer.mozilla.org/en-US/docs/Web/API/Event/target) |
-| Parameters | 매개변수 | [링크](http://localhost:5042/en-US/docs/Web/API/AudioWorkletNode/AudioWorkletNode#parameters) |
-| Value | 값 | [링크](https://developer.mozilla.org/en-US/docs/Web/API/Event/target) |
+| Compatibility notes | 호환성 참고 사항 | <https://developer.mozilla.org/en-US/docs/Web/API/Event/target> |
+| Parameters | 매개변수 | <http://localhost:5042/en-US/docs/Web/API/AudioWorkletNode/AudioWorkletNode#parameters> |
```

Comment:
Another localhost link, maybe markdownlint should check for `//localhost:5042`?

Reply:
```suggestion
| Parameters | 매개변수 | <https://developer.mozilla.org/en-US/docs/Web/API/AudioWorkletNode/AudioWorkletNode#parameters> |
```